docs/about/migration/23-01.ipynb

```diff
@@ -0,0 +1,193 @@
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a3501a6-e855-4667-b2fc-a10a5a8bb669",
+   "metadata": {},
+   "source": [
+    "An operation between `var` and `norm` will raise `VariancesError` since `norm` is broadcast into dimension `'x'`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6f16545-caa4-453d-9e67-18663ca5281e",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "var / norm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0bd3114-3502-424f-a70b-ee7306264fae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.data.table_xyz(100).bin(y=3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65722b75-6aba-4a30-a9e3-2cbe1eb23394",
+   "metadata": {},
+   "source": [
+    "An operation between `binned` and `norm` will raise `VariancesError` since `norm` is broadcast to all bin elements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c33fce46-a1be-41aa-9660-c69b83a70548",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "binned / norm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9dd8781c-6003-4243-a1e3-810a4088219d",
+   "metadata": {},
+   "source": [
+    "## Working around the new behavior\n",
+    "\n",
+    "Scipp will not be able to support an automatic approach for handling these cases correctly, since this would be either very costly or very problem-specific.\n",
+    "An easy **but wrong** work around (equivalent to the old behavior in almost all cases) is to set the variances of the term being broadcast to `None`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "212a76f9-18b0-4390-a557-3c9cdd251c69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "stream_handler = logging.StreamHandler()\n",
+    "logger = sc.get_logger()\n",
+    "logger.addHandler(stream_handler)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "656d28de-d695-4523-a1c4-269853aa94ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger.warning(\n",
+    "    \"\"\"ATTENTION:\n",
+    "    Statistical uncertainties of the normalization term were IGNORED.\n",
+    "    This is probably incorrect.\"\"\"\n",
+    ")\n",
+    "norm.variances = None\n",
+    "var / norm\n",
+    "binned / norm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab2df54b-0af1-4cbd-96eb-cabae9247f0b",
+   "metadata": {},
+   "source": [
+    "We repeat that this is **incorrect**.\n",
+    "We strongly recommend to issue an explicit warning to the user, as shown in the example."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "800f1743-9031-4923-9712-1ab9e703f505",
+   "metadata": {},
+   "source": [
+    "## Fixing the problem\n",
```

Comment:
Mention resampling methods?

Reply:
What are you referring to?

Reply:
Using bootstrap or jacknife if an analytical derivation is not possible.

Reply:
I don't think this is a fix, as it takes 1000x longer?

Reply:
And an analytical derivation may be impossible. The truth is that this problem often cannot be solved.

Reply:
Isn't that what I am saying?